pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -32,28 +48,28 @@ in stdenv.mkDerivation rec {
     sha256 = "1xl7z0vwbn5iycg7amka9jd6hxd8nmfk7nahi4p9w2bnw9f0wcrl";
   };
 
+  ARROW_MIMALLOC_URL = fetchurl {
```

Comment:
So jemalloc is not needed now?

Reply:
See https://arrow.apache.org/docs/cpp/api/memory.html?highlight=jemalloc#memory-pools.
jemalloc is still the default, but it is possible to use mimalloc as well. User code gets to choose.

mimalloc support is disabled by default, but enabling it doesn't hurt.